docs/BackupNoSQL.md

```diff
@@ -0,0 +1,153 @@
+# Backup NoSQL database guide
+
+If you use a NoSQL database or multiple databases, you must **pause** the Scalar products to create a transactionally-consistent backup. This guide explains concrete steps of how to create the transactionally-consistent backup of managed databases with the Scalar products on a Kubernetes environment. Please refer to the following document if you want to know more details on the backup way of Scalar products.
```

Comment:
Is there a particular reason for saying `the Scalar products` here and saying `ScalarDB/DL` in docs/BackupRestoreGuide.md?
I think we should use consistent naming as much as possible if we talk about the same thing.

Reply:
To reduce ambiguity, I've specified ScalarDB and ScalarDL as the target products.

```suggestion
If you are using a NoSQL database or multiple databases, you must pause the Scalar products to create a transactionally consistent backup. This guide explains how to create a transactionally consistent backup of managed databases by using ScalarDB or ScalarDL in a Kubernetes environment. 

For details on how ScalarDB backs up databases, see [A Guide on How to Backup and Restore Databases Used Through ScalarDB](https://github.com/scalar-labs/scalardb/blob/master/docs/backup-restore.md).
```

Reply:
I feel `databases that use ScalarDB or ScalarDL` is `The database utilizes some ScalarDB/ScalarDL feature`.
In other words, I feel that sentence means the ScalarDB/ScalarDL are extensions for some databases like the following image.
```
[Client] -------> [Database (with ScalarDB)]

            +---> [Database (with ScalarDL Ledger)]
            |
[Client] ---+
            |
            +---> [Database (with ScalarDL Auditor)]
```

However, in fact, the system architecture is the following.
```
[Client] -------> [ScalarDB] -----------> [Backend DB]

            +---> [ScalarDL Ledger] ----> [Backend DB]
            |
[Client] ---+
            |
            +---> [ScalarDL Auditor] ---> [Backend DB]
```

The ScalarDB and ScalarDL utilize the database as a `Backend DB` to store some data of ScalarDB or ScalarDL.
So, I think it would be better to describe `databases that are used by ScalarDB or ScalarDL` instead of `databases that use ScalarDB or ScalarDL`.

What do you think?